cse-cc-export-runner-app/src/main/java/com/farao_community/farao/cse/export_runner/app/services/MerchantLineService.java

```diff
@@ -18,16 +21,23 @@
 public class MerchantLineService {
 
     private final UctePstProcessor uctePstProcessor;
+    private final Logger businessLogger;
 
-    public MerchantLineService(MendrisioConfiguration mendrisioConfiguration) {
-
+    public MerchantLineService(MendrisioConfiguration mendrisioConfiguration, Logger businessLogger) {
+        this.businessLogger = businessLogger;
         this.uctePstProcessor = new UctePstProcessor(
                 mendrisioConfiguration.getMendrisioPstId(),
                 mendrisioConfiguration.getMendrisioNodeId());
     }
 
     public void setTransformerInActivePowerRegulation(Network network) {
-        uctePstProcessor.setTransformerInActivePowerRegulation(network);
-
+        String pstId = uctePstProcessor.getPstId();
```

Comment:
You already have the information at the level of this class of the PST ID since it is the merchanlineService which initialized the UctePstProcessor by passing it the PST ID  and the NODE ID.
I think it's not worth calling the uctePstProcessor to retrieve the PST ID information while it is this class which has already provided him with this information.

Reply:
ok I will perform the change